terra_notebook_utils/cli/table.py

```diff
@@ -60,6 +60,17 @@ def get_row(args: argparse.Namespace):
     data[f'{args.table}_id'] = e['name']
     print(json.dumps(data, indent=2))
 
+@table_cli.command("fetch-drs-url", arguments={
+    "--table": dict(type=str, required=True, help="table name"),
```

Comment:
Is it possible to fetch the drs uri without the table name?

Reply:
That would involve assumptions about table content that could be violated by user actions.